secure_message/application.py

```diff
@@ -56,10 +56,11 @@ def create_app(config=None):
     api.add_resource(MessageSend, '/message/send', '/v2/messages')
     api.add_resource(MessageModifyById, '/message/<message_id>/modify',
                      '/v2/messages/modify/<message_id>')
-    api.add_resource(MessageCounter, '/v2/messages/count')
 
     api.add_resource(ThreadList, '/threads')
     api.add_resource(ThreadById, '/thread/<thread_id>', '/v2/threads/<thread_id>')
+    api.add_resource(ThreadCounter, '/v2/messages/count',
+                     '/v2/threads/count')
```

Comment:
If the endpoint changed wouldn't we need to change r-ops? The call for count there is hitting `/v2/messages/count`

Reply:
r-ops would need to be changed but this was just to add the new endpoint alongside the current one so that both would use ThreadCount until r-ops was changed.  As it stands, I think the new endpoint should be removed (as @ajmaddaford pointed out the conflict) and any further refactoring should be separate from fixing the original bug.